### PR TITLE
Remove out of place ci.yaml file

### PR DESCRIPTION
**Pull request by llucax**

### CODE REVIEW

**Review by TalweSingh — APPROVED** [review]
Maybe add why this file is out of place and is removed in the body of the commit message.

**Comment by llucax** [comment]
If you have any comments please don't approve the PR, otherwise when auto-merge is enabled you give no chance to the PR author to actually do something about your comments. Ideally you should use the *Request changes* button if you want some changes to be made before the PR can be merged or the *Comment* button if the comments are nice-to-have/optional.

**Comment by TalweSingh** [comment]
Yea, sorry only realised this after approving. Will do next time.

**Pull request merged**
